assets/_scss/components/_forms.scss

```diff
@@ -0,0 +1,17 @@
+input.input-small {
+  @include media($tablet-up) {
+    max-width: 6rem;
+  }
+}
+
+.additional_text {
+  float: right;
+  font-style: italic;
+  font-weight: normal;
+}
+
+.usa-form-one-third {
```

Comment:
Is there a reason this can't be on the form class itself? I feel like this insinuates that you can do form-two-third, or form-one-half, but in reality you can't, which might be confusing to somebody using the library.